### PR TITLE
created booking model

### PR DESCRIPTION
**Pull request by emjbrown**
I created the Booking model and migrated the db @Tjane7 @DOGAFSR

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :cone
+
+  validates_associated :user, :cone
```
Comment (Tjane7) [inline app/models/booking.rb:5]: we dont need this one please remove it

**Review by Tjane7 — CHANGES_REQUESTED** [review]
hey you can check the comments in booking.rb :)

File: app/models/booking.rb (hunk 1 above)
Comment (emjbrown) [reply app/models/booking.rb:5]: on it!

**Review by Tjane7 — APPROVED** [review]
looks good !

**Pull request merged**